.github/workflows/codeql.yml

```diff
@@ -2,9 +2,13 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [main]
+    branches:
+      - main
+      - 'maint-*'
```

Comment:
Feel free to say no.... was wondering if we should run this on every branch which would then be:

```
branches:
  - '**'
```

Reply:
No opinion.

Reply:
I guess we can look at that later depending on if we need to stray from what is already listed.